test/new-e2e/tests/agent-shared-components/config-refresh/non_core_agents_sync_test.go

```diff
@@ -88,6 +89,12 @@ func (v *configRefreshSuite) TestConfigRefresh() {
 			agentparams.WithSecurityAgentConfig(securityAgentConfig),
 			agentparams.WithSkipAPIKeyInConfig(), // api_key is already provided in the config
 		),
+		awshost.WithAgentClientOptions(
+			agentclientparams.WithAuthTokenPath(authTokenFilePath),
+			agentclientparams.WithTraceAgentOnPort(apmCmdPort),
+			agentclientparams.WithProcessAgentOnPort(processCmdPort),
+			agentclientparams.WithSecurityAgentOnPort(securityCmdPort),
+		),
```

Comment:
Since we do not wait in the test later, do we not want to use `WithWaitForDuration` too here ?

Reply:
`WaitForDuration` has a default value, `WithWaitForDuration` is just to override it with a custom value